### PR TITLE
GitBook: Refine App Composition and Deployment guidelines

### PR DESCRIPTION
**Pull request by pwittrock**
- Also make them more clearly Alpha

### CODE REVIEW

File: docs/book/pages/app_composition_and_deployment/structure_repositories.md [hunk 1]
```diff
@@ -16,7 +19,7 @@ The are several techniques for users to structure their Resource Config files.
 | Type                                        | Summary               | Benefits                                           |
 |---------------------------------------------|-----------------------|----------------------------------------------------|
 | [Directories](structure_directories.md)        | *Simplest approach*   | Easy to get started and understand               |
-| [Branches](structure_branches.md)        | *More flexible*       | Loose coupling between release specific and operation changes |
+| **[Branches](structure_branches.md)**   | **More flexible**       | **Loose coupling between version specific and live operational changes** |
```
Comment (Liujingfang1) [inline docs/book/pages/app_composition_and_deployment/structure_repositories.md]: Why is the branch row also bold?

File: docs/book/pages/app_composition_and_deployment/structure_directories.md [hunk 2]
```diff
@@ -15,7 +17,7 @@ The are several techniques for users to structure their Resource Config files.
 | Type                                   | Summary               | Benefits                                           |
 |----------------------------------------|-----------------------|----------------------------------------------------|
 | **[Directories](structure_directories.md)**   | **Simplest approach**   | **Easy to get started and understand**               |
-| [Branches](structure_branches.md)   | *More flexible*       | Loose coupling between release specific and operation changes |
+| **[Branches](structure_branches.md)**   | **More flexible**       | **Loose coupling between version specific and live operational changes** |
```
Comment (Liujingfang1) [inline docs/book/pages/app_composition_and_deployment/structure_directories.md]: Why is the branch row also bold?

File: docs/book/pages/app_composition_and_deployment/structure_repositories.md (hunk 1 above)
Comment (pwittrock) [reply docs/book/pages/app_composition_and_deployment/structure_repositories.md]: That is a mistake.

File: docs/book/pages/app_composition_and_deployment/structure_directories.md (hunk 2 above)
Comment (pwittrock) [reply docs/book/pages/app_composition_and_deployment/structure_directories.md]: Fixed.

**Comment by monopole** [comment]
Didn't you just ask for comments on this in the sig-cli meeting?

**Comment by Liujingfang1** [comment]
@monopole I think Phil means to review and comment the kubectl book https://kubectl.docs.k8s.io/

**Pull request merged**